### PR TITLE
stratisd_cert: Add test_blockdev_set_userinfo_property

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #184

### CODE REVIEW

File: stratisd_cert.py [hunk 1]
```diff
@@ -823,6 +823,23 @@ def test_pool_set_fs_limit_too_low(self):
...
+        """
+        pool_name = p_n()
+        blockdev_path = StratisCertify.DISKS[0:1]
+        _, blkdev_paths = make_test_pool(pool_name, blockdev_path)
+
+        self._unittest_set_property(
+            blkdev_paths[0],
+            StratisDbus.BLKDEV_IFACE,
+            "UserInfo",
+            (True, dbus.String("eeeeeeeeee")),
```
Comment (mulkieran) [inline stratisd_cert.py:839]: For consistency, you should wrap the ```True``` in a ```dbus.Boolean``` constructor call, as we do w/ all other boolean values we put on the D-Bus.

**Review by mulkieran — CHANGES_REQUESTED** [review]
One minor remark about the D-Bus method call.

File: stratisd_cert.py (hunk 1 above)
Comment (bgurney-rh) [reply stratisd_cert.py]: Resolved.

File: stratisd_cert.py [hunk 2]
```diff
@@ -823,13 +823,30 @@ def test_pool_set_fs_limit_too_low(self):
             "org.freedesktop.DBus.Error.Failed",
         )
 
+    @skip(_skip_condition(1))
+    def test_blockdev_set_userinfo_property(self):
+        """
+        Test setting the UserInfo property on a block device.
+        """
+        pool_name = p_n()
+        blockdev_path = StratisCertify.DISKS[0:1]
```
Comment (mulkieran) [inline stratisd_cert.py:832]: Might as well get rid of ```blockdev_path``` identifier. Where you pass it as an argument to ```make_test_pool```, you might as well pass the value ```StratisCertify.DISKS[0:1]```

**Review by mulkieran — CHANGES_REQUESTED** [review]
One final request...

File: stratisd_cert.py (hunk 2 above)
Comment (bgurney-rh) [reply stratisd_cert.py]: Resolved.

**Comment by bgurney-rh** [comment]
Squashed into a single commit.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**